src/inversion_scripts/run_inversion.sh

```diff
@@ -71,29 +74,29 @@ fi
 
 printf "Calling postproc_diags.py, FSS=$FirstSimSwitch\n"
 if "$FirstSimSwitch"; then
-    if [[ ! -d ${SpinupDir} ]]; then
-    printf "${SpinupDir} does not exist. Please fix SpinupDir or set FirstSimSwitch to False in run_inversion.sh.\n"
-    exit 1
-    fi
-    PrevDir=$SpinupDir
+   if [[ ! -d ${SpinupDir} ]]; then
```

Comment:
It looks like the number of spaces for tab is inconsistent with above in this file

Reply:
Good catch. This is now fixed in https://github.com/geoschem/integrated_methane_inversion/pull/157/commits/c53f4d519860a678a5dec31f403770c497cfab43.